src/utils/cells.tsx

```diff
@@ -44,26 +44,32 @@ function renderProjectNameCell<T extends Project>(
   );
 }
 
-function renderLinkNameCell(name: string, id: number, route: string) {
+function renderLinkNameCell(
```

Comment:
This is used in src/modules/OrganisationsProjectsTable, but I don't see an update to its usage there in this PR.

Reply:
Cheers - fixed